src/nnet3bin/nnet3-copy-egs.cc

```diff
@@ -301,7 +340,18 @@ int main(int argc, char *argv[]) {
                 "feature left-context that we output.");
     po.Register("right-context", &right_context, "Can be used to truncate the "
                 "feature right-context that we output.");
-
+    po.Register("weights", &weight_str,
+                "Rspecifier of a table indexed by the key of input examples"
+                "and value of scalar weights."
+                "If provided, the supervision output in eg is scaled using weight."
+                " Scaling output supervision is the same as scaling the "
+                "derivative during training for linear objective."
+                "The default is one, which means we are not applying per-example weights.");
+    po.Register("outputs", &output_str,
```

Comment:
@danpovey  any final idea? since it requires changing lots of scripts for option level change!